kod/object/item/passitem/chalice.kod

```diff
@@ -26,9 +26,18 @@ resources:
    chalice_cant_get = "You try to pick up the chalice, but it mystically clings to the stone altar."
 
    chalice_use = "You sip a bit of the water out of the Chalice of Rain."
-   chalice_empty = "You sip the last bit of water from the Chalice of Rain."
+   chalice_empty = "The Chalice of Rain crumbles to dust in your hand as you sip the last bit of water from it."
    chalice_combine = "You pour the water from one Chalice into the other."
    chalice_cant_use_pkill = "Only innocents may partake of the chalice."
+   chalice_refilled = "The Chalice of Rain glows faintly for a moment, renewed by the spirit of Shal'ille that permeates the area."
+   chalice_refilled_sound = water_scoop01.wav
+
+   chalice_condition_exc = " is filled to the brim with pure water."
+   chalice_condition_exc_mended = " is filled to the brim with pure water."
```

Comment:
These two strings are the same?

Reply:
Yeah.  Chalices can't be mended so having alternate text for edge cases like dm-created chalices with only 1 or 2 max hits doesn't seem necessary.  I can change it to something different if you want, but players are unlikely to ever see it under normal circumstances.